scaling_performance/optimizing_storage.adoc

```diff
@@ -173,8 +198,10 @@ Application use cases vary from application to application, as described in the
 
 * Storage technologies that support dynamic PV provisioning have low mount time latencies, and are not tied
 to nodes to support a healthy cluster.
-* NFS does not guarantee read-after-write consistency and is not recommended for applications which require it.
-* Applications that depend on writing to the same, shared NFS export may experience issues with production workloads.
+* Application developers are responsible for knowing and understanding the storage
```

Comment:
Often, Application Developers have little or no say in their storage backends.  They take what they are given, for better or for worse.  It also intends to force Application Developers to learn about storage.  A worthy cause but maybe not always an achievable goal.

As such, this change muddles what the actual recommendations are.